Doc/library/subprocess.rst

```diff
@@ -797,6 +800,32 @@ on Windows.
       :data:`SW_HIDE` is provided for this attribute. It is used when
       :class:`Popen` is called with ``shell=True``.
 
+   .. attribute:: lpAttributeList
+
+      A dictionary of additional attributes for process creation as given in
+      ``STARTUPINFOEX``, see
+      `UpdateProcThreadAttribute <https://msdn.microsoft.com/en-us/library/windows/desktop/ms686880(v=vs.85).aspx>`__.
+
+      Supported attributes:
+
+      **handle_list**
+         Sequence of handles that will be inherited. *close_fds* must be true if
+         non-empty.
+
+         The handles must be made inheritable by
```

Comment:
You could say "The handles must be temporarily made inheritable by `os.set_handle_inheritable` when passed to the `Popen` constructor, else `OSError` will be raised...". Then the second sentence can be removed.

The second paragraph I think should be a warning. How about this?

       .. warning::
       
          In a multithreaded process, use caution to avoid leaking handles
          when combining this feature with concurrent calls to other process
          creation functions that inherit all handles such as
          :func:`os.system`.  This also applies to standard handle
          redirection, which temporarily creates inheritable handles.